horovod/common/operations.cc

```diff
@@ -228,7 +228,7 @@ struct HorovodGlobalState {
 
 // We reuse CUDA events as it appears that their creation carries non-zero cost.
 // Event management code is only used in NCCL path.
-#if HAVE_NCCL || HAVE_DDL
+#if HAVE_NCCL || HAVE_DDL || HAVE_CUDA
```

Comment:
You can simplify to just `HAVE_CUDA`

Reply:
Makes sense, wasn't sure of that. Changed now.